docs/premium-features/break-the-monolith.md

```diff
@@ -8,51 +8,82 @@ pagination_prev: null
 
-Overview mode is the view-only default mode for viewing your services and entities.
+Overview Mode introduces several components to ease navigating throughout the interface.
+
+### Service Filter
+
+On the upper left-side of the Architecture tab is the filter component and the associated search bar.
+
+Each individual colored button with an icon represents one of your services. _Hover_ over the button to see the name of your service. _Click_ the button to hide/show the service.
 
-#### Redesign Mode
+The associated search field helps you search through your services by name.
 
-Redesign mode allows you to redesign and restructure services, facilitating the movement of entities between them.
+The filter is useful if you only need to break specific services.
+
+### Drag-and-Drop Interface Management Buttons
+
+Directly below the Service Filter are various buttons to help you manage the drag-and-drop interface.
+
+_Hover_ over each button to see the name of what the button does.
+
+1. **Toggle Field List** toggles the associated field names for all the entities in your service.
+2. **Reset Layout** resets the Zoom level back to normal.
+3. **Zoom In** increases the Zoom level of the interface.
+4. **Zoom Out** decreases the Zoom level of the interface.
+
+## Redesign Mode
+
+Redesign mode allows you to restructure services by facilitating the movement of entities between them using our drag-and-drop interface.
+
+Click on the **Redesign** button to enter Redesign Mode and select a service to redesign.
 
 :::note
-**Free users** have access to Overview mode _without_ the Redesign capabilities.
+You can currently redesign only **one** service at a time.
 :::
 
-![Amplication's Break The Monolith View Mode](./assets/break-the-monolith-view-mode.png)
+You can move entities between services in two ways:
 
-### Step 2: Manual Service Redesign
+1. Drag and drop entities into the appropriate service.
+2. Select multiple entities using the dedicated menu and move them in bulk to create micro-services.
 
-After entering Redesign Mode, manually move entities between services. You can select multiple entities using the dedicated menu and move them in bulk to create micro-services.
+![Amplication's Break The Monolith Redesign Mode](./assets/break-the-monolith-redesign-mode.png)
 
-To migrate entities to other services, you can drag and drop them into the appropriate service. After making changes, you can click on the **Apply** button to implement those changes.
+Redesign mode introduces a few more buttons to help you while redesigning your service.
 
-![Amplication's Break The Monolith Redesign Mode](./assets/break-the-monolith-redesign-mode.png)
+1. **Add New Service** creates a new service so that you can drag-and-drop entities between 
+2. **Fetch Updates From Server** gets any updates from the server. This is useful in case updates were made during the monolith update process.
```

Comment:
```suggestion
2. **Fetch Updates From Server** gets any updates from the server. This is useful in case updates were made during the monolith redesign process.
```